src/create-select.tsx

```diff
@@ -167,6 +167,7 @@ const createSelect = (props: CreateSelectProps) => {
         } else {
           if (focusedOptionIndex() > -1) setFocusedOptionIndex(-1);
           clearInputValue();
+          hideInput();
```

Comment:
When we have `multiple` to `true` we sometimes want the cursor to remain visible for entering more text for selecting the next option. Adding this here will hide the cursor in that case, so we don't want to do that.